src/app/CASEClient.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/CASEClient.h>
+
+namespace chip {
+
+CASEClient::CASEClient(const CASEClientInitParams & params) : mInitParams(params) {}
+
+CHIP_ERROR CASEClient::EstablishConnection(PeerId peer, const Transport::PeerAddress & peerAddress, uint32_t mrpIdleInterval,
```

Comment:
For consistency, suggest:

```suggestion
CHIP_ERROR CASEClient::EstablishSession(PeerId peer, const Transport::PeerAddress & peerAddress, uint32_t mrpIdleInterval,
```